schemas/2019-04-01/deploymentTemplate.json

```diff
@@ -1278,14 +1278,6 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2016-10-01/Microsoft.StorSimple.json#/resourceDefinitions/managers_storageAccountCredentials" },
                   { "$ref": "https://schema.management.azure.com/schemas/2016-10-01/Microsoft.StorSimple.json#/resourceDefinitions/managers_storageDomains" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-10-01/Microsoft.SignalRService.json#/resourceDefinitions/SignalR" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2017-08-15/Microsoft.NetApp.json#/resourceDefinitions/netAppAccounts" },
```

Comment:
Are the ANF resource types not needed in this version of the dep,oymentTemplate.json?

Reply:
They've been moved into `schemas/common/autogeneratedResources.json`, which is referenced by this file. So they'll still appear in the list of resources.